README.md

```diff
@@ -96,3 +96,104 @@ So, if you want to distribute a Forge port of Iris, we'd prefer if you let us kn
+
+• Complementary Shaders (https://www.curseforge.com/minecraft/customization/complementary-shaders)
+
+• Prismarine Shaders (https://www.curseforge.com/minecraft/customization/prismarine-shader)
+
+• BSL Shaders (https://bitslablab.com/bslshaders/)
+
+• Continuum 1.3 
+
+• Continuum 2.0.4
+
+• SEUS V11 (https://www.sonicether.com/seus/)
+
+• SEUS Renewed (https://www.sonicether.com/seus/)
+
+• Sildur's Enhanced Default (https://sildurs-shaders.github.io/)
+
+• Sildur's Vibrant Shaders (https://sildurs-shaders.github.io/)
+
+• Skylec v4.0 (https://www.curseforge.com/minecraft/customization/skylec-shader)
+
+• Tea V1.5.3
+
+• XorDev's shaders
+
+• RedHat shaders (https://www.curseforge.com/minecraft/customization/redhat-shader-v1-chocapic13-edit)
+
+The following shaders have been tested and work with some minor issues:
+
+• Project Luma [No Sky, broken end portals/gateways]
+
+• Triliton V8 [reflections broken]
+
+• Stereo's Default+ [No Sky on 1.17] 
+
+> Q. Will (insert feature) part of OptiFine be added to iris? 
+
+A. Iris is specifically a shaders mod. Other features are not planned for Iris. However, many other wonderful mods have been made by the fabric modding community which cover many of the features of OptiFine. 
+
+The following is a short list, more alternatives can be found through further research, the list is alphabetical.
+
+Better Grass/Snow - https://www.curseforge.com/minecraft/mc-mods/lambdabettergrass
+
+Chunk Caching - https://www.curseforge.com/minecraft/mc-mods/bobby
+Bobby is incompatible with Starlight.
+
+Connected Textures - https://modrinth.com/mod/continuity
+
+Custom Item Textures - https://modrinth.com/mod/cit-resewn 
+You need to enable “Broken Paths” or MANY PACKS WILL NOT WORK. 
+
+Dynamic Lights - https://www.curseforge.com/minecraft/mc-mods/lambdynamiclights
+
+Smart Leaves - https://www.curseforge.com/minecraft/mc-mods/cull-leaves 
+
+Performance - Iris is already bundled with Sodium but we also recommend Lithium (https://modrinth.com/mod/lithium), Hydrogen (https://modrinth.com/mod/hydrogen), and FerriteCore (https://www.curseforge.com/minecraft/mc-mods/ferritecore-fabric). You can also use either Phosphor (https://modrinth.com/mod/phosphor) or Starlight (https://modrinth.com/mod/starlight). Phosphor does not have a 1.17 version.
+
+Various OptiFine features including toggles for animations, particles, and fog - https://www.curseforge.com/minecraft/mc-mods/sodium-extra
+
+Zoom - https://www.curseforge.com/minecraft/mc-mods/ok-zoomer or https://www.curseforge.com/minecraft/mc-mods/logical-zoom, https://modrinth.com/mod/spyzoom for a vanilla alternative
+
+You will also need https://modrinth.com/mod/indium/ and https://www.curseforge.com/minecraft/mc-mods/fabric-api for most mods as well as https://www.curseforge.com/minecraft/mc-mods/modmenu for settings.
+
+If you need further help please ask our helpers and mods in iris-support, and someone will be there shortly to assist you! 
```

Comment:
This doesn't make any sense to have in GitHub.

Reply:
Fair